core/test/lib/stack-packs-test.js

```diff
@@ -260,6 +260,7 @@ Array [
       Array [
         "unminified-warning",
         "disable-bundling",
+        "preload-lcp-image",
```

Comment:
This is a bug right?

Reply:
it's because there are stack packs for `preload-lcp-image` so they move to `unrecognizedKeys`

Reply:
updated stack pack ids in https://github.com/GoogleChrome/lighthouse-stack-packs/pull/89